api/test/test_helper/fake_android_plugin.gd

```diff
@@ -24,11 +24,28 @@
 class_name FakeAndroidPlugin
 extends Object
 
+signal removePackSuccess(signalID)
```

Comment:
Should these be separated by underscores rather than camelCase?

Reply:
This camelCase naming for mock signals/functions in fake_android_plugin is intentional, since we are mocking existing functions and signals written in Java.